packages/components/src/utils/useShortcut.ts

```diff
@@ -0,0 +1,34 @@
+import { useCallback, useEffect, useLayoutEffect, useRef } from 'react';
+
+export const useShortcut = (key: string, callback: () => void, modifiers?: { ctrl?: boolean; alt?: boolean; shift?: boolean }) => {
+  const callbackRef = useRef(callback);
+  useLayoutEffect(() => {
+    callbackRef.current = callback;
+  });
+
+  const handleKeyPress = useCallback(
+    (event: KeyboardEvent) => {
+      const ctrl = modifiers ? modifiers.ctrl : true;
```

Comment:
`const ctrl = modifiers?.ctrl ?? true;`

Reply:
It's not the same thing in this case.
If no `modifiers` object is provided I want the defaults to be used (`true`, `true`, `false`).
If a `modifiers` object is provided, I want to use the values it contains or `false` for missing values.

With the current implementation `modifiers = {shift: true}` results in: `{ctrl: false, alt: false, shift: true}`
With your change it would result in: `{ctrl: true, alt: true, shift:true}` because it's going to use the defaults for the missing values.

Of course we could change the behaviour but I think this makes more sense or make the properties of modifiers all required in the first place.